src/components/PopupConfirm.js

```diff
@@ -0,0 +1,30 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector("#popup-confirm-form");
+    this._submitButton = this._popupElement.querySelector(".modal__button");
+  }
+
+  setSubmitAction(action) {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.